articles/connections/nativesocial/facebook.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Add Facebook Login to Native Apps
+description: Learn how to add login functionality to your native app with Facebook. 
+topics:
+  - authentication
+  - connections
+  - social
+  - facebook
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Facebook Login to Native Apps
+
+You can add functionality to your native application to allow your users to authenticate using Facebook natively, within the application. This does not require redirection via a web browser.
+
+## How it works
+
+The Native Facebook login flow works as follows:
+
+* **Step 1**: The application authenticates a user via the Facebook SDK and acquires an Access Token.
+* **Step 2**: The application uses that Access Token to request a special [Facebook Session Info Access Token](https://developers.facebook.com/docs/facebook-login/access-tokens/session-info-access-token).
+* **Step 3**: The application can then use the Facebook Session Info token to request a user profile from Auth0.
```

Comment:
Step 3: Use the Facebook SDK to retrieve the users's profile
Step 4: The application can then use the Facebook Session Info token to authenticate with Auth0.